protelis/protelis-interpreter/src/main/java/org/protelis/lang/ProtelisLoader.java

```diff
@@ -167,7 +185,7 @@ private ProtelisLoader() {
      * @throws IllegalArgumentException
      *             when the program has errors
      */
-    public static ProtelisProgram parse(final String program) {
+    public static ProtelisProgram parse(final String program) throws IllegalArgumentException {
```

Comment:
Remove the useless `throws`: `RuntimeException`s only need to get documented ad the Javadoc level, they should not be declared in a throws block.

Reply:
fixed